src/app/pwa/(pages)/profile/_components/onramps/onramp-stripe.tsx

```diff
@@ -26,8 +26,8 @@ interface OrampWithStripeProps extends React.HTMLAttributes<HTMLDivElement> {
     appearance?: StripeOnrampAppearance
 }
 
-if (!process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_TEST) {
-    throw new Error('NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_TEST not set')
+if (!process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY) {
```

Comment:
If we check the key existence here, we should do the same for STRIPE_SECRET_KEY in the route, to keep consistency.

but don't worry, I will implemente a better environment check that I used in another project, with t3 env tiny library.